packages/amplify-graphql-api-construct-tests/package.json

```diff
@@ -17,20 +17,28 @@
   ],
   "private": true,
   "scripts": {
-    "e2e": "jest --verbose --forceExit",
-    "build-tests": "tsc --build tsconfig.tests.json"
+    "e2e": "yarn test:e2e",
+    "build-tests": "tsc --build tsconfig.tests.json",
+    "test:e2e": "jest --verbose --forceExit",
+    "test": "jest --verbose --forceExit src/__tests__/unit-tests"
```

Comment:
Added a `test` target to run the existing unit tests that were previously only being run during E2E runs.